src/queries/team.py

```diff
@@ -101,3 +101,38 @@ async def remove_team(team_name: str):
     await aconn.commit()
     await acur.close()
     await aconn.close()
+
+
+async def increase_puzzles_solved(team_name: str):
+    aconn = await psycopg.AsyncConnection.connect(DATABASE_URL)
+    acur = aconn.cursor()
+
+    # this should never run since the team name should always be
+    # valid when calling this function
+    if not await get_team(team_name):
+        return False
+
+    # get the current number of solved puzzles by the team
+    await acur.execute(
+        """
+        SELECT t.puzzle_solved
+        FROM public.teams AS t
+        WHERE t.team_name = %s
+        """,
+        (team_name,),
+    )
+
+    puzzles_solved = (await acur.fetchone())[0]
+
+    await acur.execute(
+        """
+        UPDATE public.teams
+        SET puzzle_solved = %s
+        WHERE team_name = %s
+        """,
+        (puzzles_solved + 1, team_name),
+    )
```

Comment:
You can do something like this :D

```suggestion
    await acur.execute(
        """
        UPDATE public.teams
        SET puzzle_solved = puzzle_solved + 1
        WHERE team_name = %s
        """,
        (puzzles_solved + 1, team_name),
    )
```

Reply:
except i wouldn't have the puzzles_solved + 1 in the tuple, right

Reply:
oh wait yup, good catch